packages/notary/actors/notary_actor.py

```diff
@@ -38,21 +54,29 @@ def get(self, key, schema_out):
         ```
         """
         out = schema_out.new()
-        # TODO: return json object
-        out.message = "your key is {}".format(key)
+        bcdb = self.bcdb_get()
+        for model in bcdb.get_all():
+            if model.key == key:
+                return model
+
+        out.message = "this key doesn't exist"
```

Comment:
This should return an error not just a message. What is the proper way to return error to the client in gedis ?
Raise an exception ?